specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
So then maybe a CVR should not contain a `ballot_manifest_info`, but should contain those other essential fields (the ones it ships with in the CSV file). "ImprintedId", I thought, was just "TabulatorNum-BatchId-RecordId" - that's what it appears as in the CSV files, and I'd imagine it would be fine to just index on that. Why do you think it's problematic?